realsense2_camera/src/base_realsense_node.cpp

```diff
@@ -286,13 +286,13 @@ sensor_msgs::msg::Imu BaseRealSenseNode::CreateUnitedMessage(const CimuData acce
     rclcpp::Time t(gyro_data.m_time_ns);  //rclcpp::Time(uint64_t nanoseconds)
     imu_msg.header.stamp = t;
 
-    imu_msg.angular_velocity.x = gyro_data.m_data.x();
-    imu_msg.angular_velocity.y = gyro_data.m_data.y();
-    imu_msg.angular_velocity.z = gyro_data.m_data.z();
+    imu_msg.angular_velocity.x = -gyro_data.m_data.z();
```

Comment:
no need to swap axes, these gyro frame is in ROS CS, and IMU frame should be in same coordinates.
this is not the optical one, which we publish later in other function.